src/main/java/uk/gov/justice/laa/crime/cfecrime/controllers/exception/ErrorResponse.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.laa.crime.cfecrime.controllers.exception;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.http.HttpStatus;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class ErrorResponse {
+    private HttpStatus statusCode;
+    private String message;
+    public ErrorResponse(String message){
```

Comment:
don't think this constructor should exist as it doesn't set the status code. Think all of the instance variables should be final